### ISSUE TITLE
[1.16.5] Charm recipes not tweakable for custom packs

### ISSUE THREAD

**Issue by Freebi85** [P3pp3rF1y/Reliquary#534]
Hey there^^

I'm trying to remove all the charm recipes in order to replace them with custom ones for an expert pack.

The issue is that you can't remove any of the standard charm recipes at all. 
Neither with Crafttweaker nor Kubejs. There is also no config for this. Would it be possible to set up something like this?

Would be great 😃 

Cheers 🍺

**Comment by P3pp3rF1y** [comment]
I will need more info on that - there's `xreliquary:mob_charm` recipe which even though it's custom behind the scenes it gets registered as any other recipe so it should be possible to just remove it. Are you not seeing it in those tools?

**Comment by Freebi85** [comment]
Yeah i tried with the usual functions to remove it in kubejs by output, by id, by regex and in CT with the usual removal methods but it doesent work at all^^ But it works with items like shear od the winter.

Kubejs
event.remove({output: 'xreliquary:mob_charm'})
event.remove({id: '/xreliquary:mob_charm_\\w+/'})

CT
craftingTable.removeRecipe(<item:xreliquary:mob_charm>);

**Comment by Freebi85** [comment]
Hey :) Do you need still more info on that? 😄

**Comment by P3pp3rF1y** [comment]
I am no expert here, but as far as I can see there's only one recipe in Reliquary with id `xreliquary:mob_charm` so you don't seem to be removing correct one perhaps? As for searching by anything else than id I am guessing that it doesn't work in these tools for custom recipes as they can't have the understanding what data go in custom recipes and how it's named.

Sorry that's all I can help with here as I am not expert on them and don't really have time to get involved in the tools. I actually feel people who support them should be able to answer these questions pretty easily if you show them the recipe. 
And with that I am closing this now.

**Comment by Freebi85** [comment]
Yeah i tried lot of things but this particular one just dont want to be changed at all^^ 
I also tried to just override the mob_charm datapack but its still always the same recipe with fragments, leather and a string no matter what. I guess it has something to do with recipe type since the mobcharm is
  `"type": "xreliquary:mob_charm",`
instead of
  `"type": "minecraft:crafting_shaped", `
  like the other ones. 
  
Oh well no biggie, will find another way^^ Thanks for your time tho. Cheers 🍺

**Comment by copygirl** [comment]
I want to mention I've also tried to remove the mob charm, belt and spawn egg recipes and it's not possible. The only recipes that can be removed are the mob charm fragment ones. Here is my KubeJS script:
```js
onEvent("recipes", event => {
  event.remove({ output: "reliquary:mob_charm_belt" });
  event.remove({ id: /^reliquary:mob_charm/ });
  event.remove({ id: /^reliquary:mob_charm_fragments/ }); // This works, I think.
  event.remove({ id: /^reliquary:reliquary\.fragment_to_spawn_egg/ });
});
```
This issue causes usage lookups for recipes involving eggs, leather and string to be drowned in a couple pages of mob charm related items which I intended to disable in my modpack. None of these items are obtainable, so it would be nice to have them and their recipes hidden from the player.

This is on version `1.18.2-2.0.8.1139`.
